Micromouse/Maze.cpp

```diff
@@ -56,6 +56,14 @@ bool Maze::isWall(CellCoordinate pos, Direction dir) const {
     return isWall(pos.toNode() + dir);
 }
 
+bool Maze::isWall(NodeCoordinate pos, Direction dir) const {
```

Comment:
Not a huge issue, but is this really necessary? `isWall(pos, dir)` isn't really shorter than `isWall(pos + dir)`.